akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -43,6 +43,8 @@ class HttpExt(private val config: Config)(implicit val system: ActorSystem) exte
 
   override val sslConfig = AkkaSSLConfig(system)
 
+  private[this] val defaultConnectionPoolSettings = ConnectionPoolSettings(system)
```

Comment:
As @rkuhn said, we'll have to make those behave like their previous case class counterparts (broken since moving to the "easier to binary compat evolve" PR).